src/sentry/integrations/slack/utils/escape.py

```diff
@@ -14,3 +16,16 @@ def escape_slack_text(txt: str | None) -> str:
     if not txt:
         return ""
     return txt.translate(translator)
+
+
+def escape_slack_markdown_text(txt: str | None) -> str:
```

Comment:
is this function potentially dangerous if the string has a code block in it ("```")?

Reply:
Oh I see your GH comment
> This is because we indicate the start of the codeblock with 3 backticks, so if 3 backticks appear in a row we will exit out of the code block and the rest of the text will be rendered as normal text.

Reply:
yes it is, which is why i'm just gonna reduce it to one backtick. plus code blocks won't get re-rendered in a code block 😅